.github/workflows/build-package-docs.yaml

```diff
@@ -0,0 +1,95 @@
+name: Ansible package docs build
+on:
+  workflow_dispatch:
+    inputs:
+      github_fork:
+        description: 'GitHub Fork'
```

Comment:
I realized that it only says "fork" but in fact isn't. Instead, this input represents an account/org that has a repository called `ansible-documentation` exactly. So it should probably be called called a "ansible-documentation repository fork owner".